imports/plugins/included/product-variant/client/templates/products/productGrid/content.html

```diff
@@ -2,6 +2,16 @@
   <div class="grid-content">
     <a href="{{pathFor 'product' handle=handle}}" data-event-category="grid" data-event-action="product-click" data-event-label="grid product click" data-event-value="{{_id}}">
       <div class="overlay">
+        {{#each ratings 4}}
```

Comment:
can you explain what this is, I mean the hardcoded 4

Reply:
The hardcoded 4 is just a placeholder, as there are no ratings in the product schema right now.